nucliadb/nucliadb/ingest/orm/knowledgebox.py

```diff
@@ -213,11 +215,17 @@ async def create(
             logger.error(f"{uuid} KB could not be created")
             failed = True
 
+        if release_channel is None:
+            release_channel = ReleaseChannel.STABLE
```

Comment:
nit: is there any reason why not setting the default value in the argument of the function definition?